bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
+
+MAIL_TO = ["fsfod11@gmail.com"]
```

Comment:
Do you want your email address in the sources?

Reply:
well its just for the bencher machines this file